src/formatDate.js

```diff
@@ -51,6 +51,32 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const dateArray = date.split(fromFormat[fromFormat.length - 1]);
+  const dateObject = {};
+
+  for (let i = 0; i < fromFormat.length - 1; i++) {
+    dateObject[fromFormat[i]] = dateArray[i];
+  };
+
+  if (!('YY' in dateObject)) {
+    dateObject['YY'] = dateObject['YYYY'].slice(2);
+  };
+
+  if (!('YYYY' in dateObject)) {
+    if (dateObject['YY'] < '30') {
```

Comment:
Make sense to use ternary operator instead of if else as it will make a code shorter